recipe/meta.yaml

```
@@ -28,8 +30,9 @@ outputs:
       script: python -m pip install . --no-deps --ignore-installed -vv
```

I think this is needed in every `build` section
```suggestion
      script: python -m pip install . --no-deps --ignore-installed -vv
      noarch: python
```

---

Yeah, took a look at a couple of feedstocks and it looks like it doesn't inherit, ok...